great_expectations/rule_based_profiler/config/base.py

```diff
@@ -411,6 +410,21 @@ def get_config_class(cls) -> Type["RuleBasedProfilerConfig"]:  # noqa: F821
     def get_schema_class(cls) -> Type["RuleBasedProfilerConfigSchema"]:  # noqa: F821
         return RuleBasedProfilerConfigSchema
 
+    @classmethod
+    def from_commented_map(
+        cls, commented_map: CommentedMap
+    ) -> "RuleBasedProfilerConfig":
+        try:
+            config: dict = cls._get_schema_instance().load(commented_map)
+            config.pop("class_name", None)
+            config.pop("module_name", None)
```

Comment:
Overrode the parent method because we need to pop these values. Could this be done in a cleaner fashion?